dts/arm/st/g0/stm32g0.dtsi

```diff
@@ -111,6 +111,16 @@
 			};
 		};
 
+		rtc: rtc@40002800 {
+			compatible = "st,stm32-rtc";
+			reg = <0x40002800 0x400>;
+			interrupts = <2 0>;
+			clocks = <&rcc STM32_CLOCK_BUS_APB1 0x10000400>;
```

Comment:
Please enable PWR in clock control's `config_enable_default_clocks` as this is done for other series.
Also applies to G4

Reply:
according to the ref man of the stm32G0, the access to the backup domain is similar to stm32G4, consequently, the same sequence is expected for both series: 
1. enable the power interface clock
2.  set the DBP bit in the Power Control register 1 (PWR_CR1)